### ISSUE TITLE
Missing EPSG role in output HNAP XML

### ISSUE THREAD

**Issue by alexandreleroux** [geopython/pygeometa#22]
Possible bug, at least a feature request for HNAP2.3 compliance.

In current HNAP XML output, for the EPSG we have:
```
    <gmd:role gco:nilReason="missing"/>
```

while ECDC expects something along the lines of:
```
<gmd:role xmlns:srv="http://www.isotc211.org/2005/srv" xmlns:napec="http://www.ec.gc.ca/data_donnees/standards/schemas/napec">
  <gmd:CI_RoleCode codeList="http://nap.geogratis.gc.ca/metadata/register/napMetadataRegister.xml#IC_90" codeListValue="RI_413">originator; auteur</gmd:CI_RoleCode>
</gmd:role>
```
In this example, I chose 'Originator' as role, however, the ECDC proposes 15 possible values, such as  Author, Distributor, Editor, Owner, Publisher, Resource provider, Rights holder, etc.

Let me know how I can help, thanks -- Alex

**Comment by tomkralidis** [comment]
Fixed in master branch via #28

**Comment by alexandreleroux** [comment]
confirmed fixed, thanks
